### DISCUSSION TITLE
希望绳网的UI可以优化一下

### DISCUSSION THREAD

**Discussion by canyusuo** [share121/inter-knot#4629] [灌水]
怎么说呢，感觉就是“米”味不足
![9a7ni7sr](https://github.com/user-attachments/assets/3a1ecb6a-8f6b-4cbd-88b2-9d2461f585bf)

**Comment by LMA936** [comment]
因为这个站就是复刻的绝区零绳网呀
<img src="https://upload-bbs.miyoushe.com/upload/2022/12/08/a2fa805a0a782cdb8ab44ddaf8d5343d_7408497524870727211.png" alt="阿姬倒地" width="75" >

**Reply by canyusuo** [reply]
这个UI界面跟游戏里的还是不太一样的，以前版本的绳网跟游戏里才一样，但听说不好用。但现在的UI确实不太好看，所以希望可以优化一下
